### DISCUSSION TITLE
Version 0.6.0 breaks my project

### DISCUSSION THREAD

**Discussion by Kokovec** [sekigon-gonnoc/Pico-PIO-USB#121] [Q&A] [answered]
I've been working on a project using the Adafruit Feather RP2040 with USB Type A Host.
When I upgraded from 0.5.3 to 0.6.0 it fails to enumerate.
Downgrading to 0.5.3 fixes it.
Are there any configuration or dependencies that I need to be aware of?

**Comment by ghost** [comment, marked as answer]
Hi @Kokovec,
there is a completely new approach to the TX routine in the new version. Thus  only one PIO block is used instead of two. On the other hand the main processor has a little bit more work to do.
If you don't need the remaining PIO for something else, it is not a disadvantage to use the previous version. (my opinion)

**Reply by Kokovec** [reply]
In that case I'll keep using the previous version.

**Comment by Kokovec** [comment]
v0.6.1 fixes the enumeration issue.
However, the NeoPixel library doesn't function while using this version.
I have gone back to (0.5.3).
